modules/modules.properties

```diff
@@ -196,14 +196,11 @@ bundle.version[com.liferay.asset.categories.test]=1.0.0
 bundle.version[com.liferay.asset.category.property.api]=3.0.0
 bundle.version[com.liferay.asset.category.property.service]=3.0.0
 bundle.version[com.liferay.asset.category.property.test]=1.0.0
-bundle.version[com.liferay.asset.display.api]=3.0.0
```

Comment:
@dantewang 
Some of the changes are not related to this pull. Could you please make it clear what should be updated with your change? Please resend.

Thanks.
Tina.